src/types/config.ts

```diff
@@ -190,6 +196,8 @@ export const MergedConfigSchema = Type.Object({
   "default-labels": Type.Array(Type.String()),
   "register-wallet-with-verification": Type.Boolean(),
   "enable-access-control": AccessControlSchema,
+  "openai-api-key": Type.String(),
+  "openai-token-limit": Type.Number(),
```

Comment:
In terms of security, `openai-api-key` shouldn't be public. right? 

@rndquu It seems the key also needed to be encrypted like the private key. do we have a simple solution to keep it secret?

Reply:
I think encrypting the `openai-api-key` is an overkill. Even if it is leaked then a new openai API key can be generated in few clicks. Simply store the `openai-api-key` in an organization wide private config repo.

Reply:
That's fine if we store it in a private repo.